level-2/index.html

```diff
@@ -360,7 +360,7 @@ <h3>
         <div class="note">
           There is an attempt to standardize the registration of websites to
           receive share data for that final use case; see <a href=
-          "https://github.com/WICG/web-share-target">Web Share Target</a>.
+          "https://github.com/w3c/web-share-target">Web Share Target</a>.
```

Comment:
This line should not be changed. Only web-share has moved, not web -share-target.

Reply:
Good catch, undone.